api/src/shop/stripe_constants.py

```diff
@@ -8,7 +8,7 @@
 # All stripe calculations are done with cents (ören in Sweden)
 STRIPE_CURRENTY_BASE = 100
 
-CURRENCY = config.get("STRIPE_CURRENCY")
+CURRENCY = "sek"  # TODO fix config.get("STRIPE_CURRENCY")
```

Comment:
You should just have to add this to your config file `.env`

```
STRIPE_CURRENCY=sek
```

Reply:
It doesn't work for some reason when I run the tests. I'm not sure why. I haven't looked much at it yet but didn't find the cause

Reply:
Found the bug and fixed it